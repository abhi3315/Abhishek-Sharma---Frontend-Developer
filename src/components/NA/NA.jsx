export default function NA() {
	return <span className='text-neutral-400'>N/A</span>;
}
