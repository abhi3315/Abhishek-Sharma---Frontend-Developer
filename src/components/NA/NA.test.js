import { render, cleanup } from '@testing-library/react';

import NA from './NA';

afterEach(cleanup);

describe('NA', () => {
	it('should render correctly', () => {
		const { container } = render(<NA />);
		expect(container).toMatchSnapshot();
	});
});
