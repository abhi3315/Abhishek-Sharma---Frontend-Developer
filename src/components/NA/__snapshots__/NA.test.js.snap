// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NA should render correctly 1`] = `
<div>
  <span
    class="text-neutral-400"
  >
    N/A
  </span>
</div>
`;
